docs/api-info.asciidoc

```diff
@@ -8,22 +8,55 @@ This lightweight endpoint is useful as a server up/down health check.
 [[api-info-endpoint]]
 === Server Information endpoint
 
-Send an `HTTP GET` request to the server information endpoint:
+This is the server information endpoint:
 
 [source,bash]
 ------------------------------------------------------------
 http(s)://{hostname}:{port}/
 ------------------------------------------------------------
 
-This endpoint always returns an HTTP 200.
+Sending an `HTTP GET` request to the server information endpoint
+will return an HTTP 200, indicating that the server is up,
+with an empty response body.
+
+Sending an `HTTP POST` request to the server information endpoint
+will return an HTTP 200, indicating that the server is up,
+with the body containing basic server information.
```

Comment:
It's not GET vs. POST that makes the difference, it's whether the request has a valid auth token.

Reply:
Oh OK, the support agent communicated that GET would only return an HTTP 200 response,
so looks like the support agent was confused by these docs also.

In my support ticket, I communicated that I am using Elastic APM 8.10.3, and that I sent an authenticated request using

```
curl --verbose --request POST --header 'Authorization: ApiKey XXX' http://localhot:8200
```

and that just returned an HTTP 200 response, and did not return a payload.

I guess that led to confusion on both my part and the part of the support agent in [case 01501264](https://support.elastic.co/cases/5008X00002UnA9UQAV)

Reply:
@axw OK, I revised this patch, and clarified the text around the examples

Reply:
Looks great, thank you :)